packages/dapp-simple-exchange/contract/simple-exchange.js

```diff
@@ -7,8 +7,8 @@ import { makeHelpers, defaultAcceptanceMsg } from '@agoric/zoe/src/contracts/hel
  * This contract is like the simpleExchange contract. The exchange only accepts
  * limit orders. A limit order is an order with payoutRules that specifies
  * wantAtLeast on one side and offerAtMost on the other:
- * [ { kind: 'wantAtLeast', units2 }, { kind: 'offerAtMost', units1 }]
- * [ { kind: 'wantAtLeast', units1 }, { kind: 'offerAtMost', units2 }]
+ * [ { kind: 'wantAtLeast', amount: amount2 }, { kind: 'offerAtMost', amount: amount1 }]
+ * [ { kind: 'wantAtLeast', amount: amount1 }, { kind: 'offerAtMost', amount: amount2 }]
```

Comment:
The new format is more like 
```
 * { offer: { 'Price', simoleans(5) }, want: { 'Asset', quatloos(3) } }
 * { offer: { 'Asset', simoleans(5) }, want: { 'Price', quatloos(3) } }
```

I don't think it makes sense to copy the comments at the top of myFirstDapp into simpleExchange. I have changes in progress that will update these comments.

Reply:
Is this contract the simple exchange or the myFirstDapp? I think it's still myFirstDapp but renamed for some reason to simpleExchange? Is that right?

Reply:
Okay, let's land those separately.